controllers/twophase/types.go

```diff
@@ -105,6 +105,22 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 
 		status.Experiment.Phase = v1alpha1.ExperimentPhaseFinished
+	} else if chaos.IsPaused() {
+		if status.Experiment.Phase == v1alpha1.ExperimentPhaseRunning {
```

Comment:
What happens when "chaos" is not "running"? For example, the current "chaos" has been "finished" or "failed".

Reply:
Rest LGTM

Reply:
> https://github.com/pingcap/chaos-mesh/pull/370#discussion_r401341634

@Yisaer 's idea is that we could pause a chaos that's not running. I think that makes some sense for the scenario in which we don't want to delete this chaos but to rerun later.